tests/python/pants_test/option/test_options_bootstrapper.py

```diff
@@ -205,6 +205,26 @@ def create_options_bootstrapper(*config_paths):
 
     self.do_test_create_bootstrapped_multiple_config(create_options_bootstrapper)
 
+
+  def test_options_pantsrc_files(self):
+    def create_options_bootstrapper(*config_paths):
+      return OptionsBootstrapper.create(args=['--pantsrc-files={}'.format(cp) for cp in config_paths])
+    with temporary_file(binary_mode=False) as fp:
+      fp.write(dedent("""
+      [resolver]
+      resolver: coursier
+      """))
+      fp.close()
+    bootstrapped_options = create_options_bootstrapper(fp.name)
+    opts_single_config = bootstrapped_options.get_full_options(known_scope_infos=[
+      ScopeInfo('', ScopeInfo.GLOBAL),
+      ScopeInfo('resolver', ScopeInfo.TASK),
+    ])
+    opts_single_config.register('', '--pantsrc-files', type=list)
+    opts_single_config.register('resolver', '--resolver')
+
+    self.assertEqual('coursier', opts_single_config.for_scope('resolver').coursier)
```

Comment:
The resolver option works in this case, although it would be more ideal to test with more unique properties, e.g. `--test-junit-jvm-options="['-Dmy.unique.property']"`, because once coursier becomes the default resolver in OSS Pants, this test may be nullified.

Reply:
this test isnt testing for the `coursier resolver` just that options are set as we expect them to, the string itself could be `foo bar` and it wouldnt matter.

Reply:
Ah gotcha. This code does not have anything to do with the task definition.